tasks/install_dist_pkgs.yml

```
@@ -133,11 +133,11 @@
     - nrinfragent_state != "absent"
     - nrinfragent_os_name|lower != 'windows'
 
-- name: setup agent service
+- name: setup agent service (windows)
   win_service:
     name: newrelic-infra
     state: "{{ nrinfragent_service_state }}"
-    enabled: "{{ nrinfragent_service_enabled }}"
+    start_mode: "{{ ( when nrinfragent_service_enabled ) | ternary('auto','manual') }}"
```

Experienced that same problem trying to install on some gcp compute windows instances and fixed with the code below.

```suggestion
    start_mode: "{{ nrinfragent_service_enabled | ternary('auto','manual') }}"
```